README.md

```diff
@@ -7,6 +7,13 @@ knowledge representation model.
 
 The project is released under GNU AGPL v3 license.
 
+
+
+
+
+
+
+
```

Comment:
Try to avoid such changes.

----

Because:
- It's equivalent to no changes.
- In this case irrelevant commit get's attached to this file.

Convention:
- Always look for file diff either on console or on browser UI.